app/repositories/application-repository.js

```diff
@@ -99,18 +99,30 @@ async function getByEmail (email) {
     })
 }
 
+/**
+ * Evaluates the sort field and direction for an application search query.
+ *
+ * @param {object} sort - An object containing the field and direction for sorting.
+ * @param {string} sort.field - The field to sort by.
+ * @param {string} [sort.direction=ASC] - The sort direction, either 'ASC' or 'DESC'.
+ * @returns {array} - An array containing the field to sort by and the sort direction.
+ */
 function evalSortField (sort) {
-  if (sort !== null && sort !== undefined && sort.field !== undefined) {
+  if (sort && sort?.field !== undefined) {
```

Comment:
```suggestion
  if (sort?.field) {
```
We need this change to avoid a sonar cloud error

Reply:
> We need this change to avoid a sonar cloud error

@alex742  added